knack/testsdk/base.py

```diff
@@ -81,8 +81,8 @@ class LiveTest(IntegrationTestBase):
 
 class ScenarioTest(IntegrationTestBase):  # pylint: disable=too-many-instance-attributes
 
-    def __init__(self, cli, method_name, filter_headers=None):
-        super().__init__(cli, method_name)
+    def __init__(self, method_name, filter_headers=None):
+        super().__init__(method_name)
```

Comment:
https://github.com/microsoft/knack/issues/287#issuecomment-2229912815 explains why positional argument `cli` is removed.